### DISCUSSION TITLE
Support for unread filter for ChannelListView?

### DISCUSSION THREAD

**Discussion by Ayush-gupta-josh** [GetStream/stream-chat-flutter#1111] [Q&A] [answered]
I want to develop an unread channels filter but based on documentation (https://getstream.io/chat/docs/flutter-dart/query_channels/?language=dart&q=Filter), there is no support for `unread_count` in filters and I can only sort the channels based on the unread count.

**Comment by imtoori** [comment, marked as answer]
hey @Ayush-gupta-josh 
Unfortunately that's not possible at the moment, it's a limitation of the backend and there's nothing we can do about it

**Reply by Ayush-gupta-josh** [reply]
Sure thanks @imtoori for the prompt reply

**Reply by TomBeckett** [reply]
@imtoori Do you know if this limitation is still in place?

**Comment by pjohn** [comment]
We would also be interested in this functionality. @imtoori if this feedback could be passed back to the backend team/product team that would be appreciated.
